website/docs/components/alert/partials/guidelines/guidelines.md

```diff
@@ -44,18 +44,20 @@ Use color logically.
 - **Warning** to help users avoid an issue. Provide guidance and actions, if possible.
 - **Critical** to indicate critical errors that need immediate action.
 
-## Critical Alert vs Critical Toast
+#### Critical alerts
 
-### When to use an Alert
-
-Use the Alert for more intrusive message communication about errors or critical disruptions at an application, page or section level where users need to take immediate action.
+Use the Alert for more intrusive message communication about errors or critical disruptions at an application, page, or section level where users need to take immediate action.
 
+!!! Do
 <Hds::Alert @type="inline" @color="critical" as |A|>
     <A.Title>You have exceeded 50 applies this month</A.Title>
     <A.Description>You may only invoke applies that destroy managed resources. Upgrade now and access additional product features, unlimited applies, and increased concurrency.</A.Description>
     <A.Button @text="Upgrade" @color="secondary" @onClick={{this.noop}} />
     <A.Link::Standalone @color="secondary" @icon="arrow-right" @iconPosition="trailing" @text="View usage" @href="#" />
   </Hds::Alert>
+!!!
+
+!!! Do
 
 Use the Alert to communicate validation errors. For more details, refer to the [form validation patterns](/patterns/form-patterns).
```

Comment:
For me this paragraph (and the others below) should live outside of the do/don't block, they're normal text